package.json

```diff
@@ -20,9 +18,19 @@
   "devDependencies": {
     "@commitlint/cli": "^9.0.1",
     "@commitlint/config-conventional": "^9.0.1",
+    "@openapi-contrib/openapi-schema-to-json-schema": "^3.0.4",
+    "@readme/eslint-config": "^3.4.2",
+    "ajv": "^6.12.4",
+    "body-parser": "^1.19.0",
     "conventional-changelog-cli": "^2.0.34",
+    "eslint": "^7.8.1",
+    "expect": "^26.4.2",
+    "express": "^4.17.1",
     "husky": "^4.2.5",
-    "lerna": "^3.22.0"
```

Comment:
Since this repo is now mono-repo and includes other languages, managing deps with Lerna no longer makes sense and I've instead built this into a `make install` target.